active_directory_ldap/tests/add_user.json

```diff
@@ -3,23 +3,23 @@
         "action": "add_user",
         "connection": {
             "host": "",
-            "port": "",
-            "use_ssl": "",
+            "port": 636,
+            "use_ssl": true,
             "username_password": {
-                "password": "",
-                "username": ""
+                "password": null,
```

Comment:
We'll need to revert this back, per request in earlier review https://github.com/rapid7/insightconnect-plugins/pull/230#discussion_r360510374